samples/Configuration files/Learn tutorials/JavaScript/NodeJS.winget

```diff
@@ -0,0 +1,34 @@
+# yaml-language-server: $schema=https://aka.ms/configuration-dsc-schema/0.2
+
+##########################################################################################################
+# This configuration will install the tools necessary to get started with Node.js development on Windows #
+# Reference: https://learn.microsoft.com/windows/dev-environment/javascript/nodejs-on-windows            #
+#                                                                                                        #
+# Note: If an existing version of Node.js is installed on the machine outside of NVM, you will be        #
+#       prompted to either allow NVM to manage that installation or remove it.                           #
+##########################################################################################################
+
+properties:
+  resources:
+    - resource: Microsoft.WinGet.DSC/WinGetPackage
+      id: Git
+      directives:
+        description: Install Git
+      settings:
+        id: Git.Git
+        source: winget
+    - resource: Microsoft.WinGet.DSC/WinGetPackage
+      id: VSCode
+      directives:
+        description: Install Visual Studio Code
+      settings:
+        id: Microsoft.VisualStudioCode
+        source: winget
+    - resource: Microsoft.WinGet.DSC/WinGetPackage
+      id: NVM
+      directives:
+        description: Install NVM for Windows
```

Comment:
```suggestion
        description: Install NVM for Windows
        securityContext: elevated
```


Even tho this installs per user, the installer requires elevation

![image](https://github.com/user-attachments/assets/fe040352-eddb-4e8b-8625-0f9f9e7fa89d)